doc/deployment-on-production-env.md

```diff
@@ -0,0 +1,43 @@
+# Deployment on Production Environment
```

Comment:
I don't feel confident giving guidance on what is the right way for production. Could we call it "arbitrary cluster"?

**Edit:** Only now I got to the arbitrary cluster guide above. Is this guide necessary at all?

Reply:
well I wanted to have a guild for ppl who just want to add kubemacpool to their cluster by just applying the kubemacpool yaml. 
maybe production env is a little bit presumptuous? what would you recommend?

Reply:
On second thought, it's redundant.. will drop this section